packages/ui/src/components/SelectInputV2/Dropdown.tsx

```diff
@@ -674,11 +683,39 @@ export const Dropdown = ({
     options,
     displayedOptions,
   } = useSelectInput()
+  const theme = useTheme()
   const [searchBarActive, setSearchBarActive] = useState(false)
   const [defaultSearchValue, setDefaultSearch] = useState<string | null>(null)
   const ref = useRef<HTMLDivElement>(null)
   const [search, setSearch] = useState<string>('')
   const [maxWidth, setWidth] = useState(refSelect.current?.offsetWidth)
+  const [marginBottom, setMarginBottom] = useState(false)
```

Comment:
Why don't we just always set the margin bottom? If the dropdown didn't reach the bottom of screen it will be an invisible margin no?